react/SlideToggle/__snapshots__/SlideToggle.test.js.snap

```diff
@@ -27,7 +27,7 @@ exports[`Slide toggle: should render a checked state 1`] = `
         />
       </div>
     </div>
-    <Text
+    <Component
```

Comment:
Looks like the way we're wiring up the HOC is breaking the display name. Probably worth looking into how we can fix this.

Looks like this breaks the docs too, if you check out the preview site: http://seek-style-guide--c749262dc5ddddd97f329a8a754138f75705374d.surge.sh/text/

Reply:
Seems that docs are not broken 🤔 or am I looking in a wrong place?

Reply:
If you look at the link above, the JSX example says `<No Display Name hero>`

Reply:
Also says this, which made me laugh:

```js
import {
  No
} from 'seek-style-guide/react';
```

Reply:
Ahahha, yeah, I see now :D Lol